### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by novykh**
Fixed infinite loop on nav tabs calculation.
+ Minor fix showed up on sentry for a missing ref access.

### CODE REVIEW

File: src/organisms/navigation/hooks/useOnTabsResize.js [hunk 1]
```diff
@@ -10,7 +12,15 @@ export default (ref, target, callback, deps) => {
...
+
+    prevValuesRef.current = {
+      prevContainerRight: containerRight,
+      prevTabWidth: tabWidth,
+    }
+
     if (tabRight >= containerRight) return callback(true)
     if (tabRight + tabWidth < containerRight) return callback(false)
-  }, [deps])
+  }, deps)
```
Comment (jacekkolasa) [inline src/organisms/navigation/hooks/useOnTabsResize.js:25]: :O

File: src/components/drops/drop/useMakeUpdatePosition.js [hunk 2]
```diff
@@ -47,6 +47,8 @@ const styles = ["top", "right", "bottom", "right", "width"]
 
 export default (target, dropRef, align, stretch) =>
   useCallback(() => {
+    if (!dropRef.current) return
```
Comment (jacekkolasa) [inline src/components/drops/drop/useMakeUpdatePosition.js:50]: 👍

**Review by jacekkolasa — APPROVED** [review]
good catch

File: src/organisms/navigation/hooks/useOnTabsResize.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import { useCallback, useRef } from "react"
 
 export default (ref, target, callback, deps) => {
+  const prevValuesRef = useRef({})
```
Comment (apostolidhs) [inline src/organisms/navigation/hooks/useOnTabsResize.js:4]: Could set outside the initial value.
`const initialRect = {prevContainerRight: -1, prevTabWidth: -1}`

**Review by apostolidhs — APPROVED** [review]

**Pull request merged**